### PR TITLE
docs: fix typos in the Config Files page

### PR DESCRIPTION
**Pull request by mrazauskas**
Few details to polish in the Config Files page.

### CODE REVIEW

File: website/docs/documentation/poku/config-files.mdx [hunk 1]
```diff
@@ -102,7 +102,7 @@ module.exports = defineConfig({
   }
 />
 
-Create a `poku.json` (or `poku.jsonc`) in your project's root directory, for example:
+Create a `.pokurc.json` (or `.pokurc.jsonc`) in your project's root directory, for example:
```
Comment (mrazauskas) [inline website/docs/documentation/poku/config-files.mdx:105]: These file names are mentioned later as default ones.

Comment (wellwelwel) [reply website/docs/documentation/poku/config-files.mdx:105]: Yes, just confirming:

https://github.com/wellwelwel/poku/blob/b23e0c5faf044d1c3ab663d7daad615a41ad3a3c/src/parsers/options.ts#L15-L18

File: website/docs/documentation/poku/config-files.mdx [hunk 2]
```diff
@@ -102,9 +102,9 @@ module.exports = defineConfig({
...
 
-```js
+```json
```
Comment (wellwelwel) [inline website/docs/documentation/poku/config-files.mdx:107]: This one is intentional, due to comments on JSONC 🙋🏻‍♂️

File: website/docs/documentation/poku/config-files.mdx [hunk 3]
```diff
@@ -102,9 +102,9 @@ module.exports = defineConfig({
   }
 />
 
-Create a `poku.json` (or `poku.jsonc`) in your project's root directory, for example:
+Create a `.pokurc.json` (or `.pokurc.jsonc`) in your project's root directory, for example:
 
-```js
+```jsonc
```
Comment (mrazauskas) [inline website/docs/documentation/poku/config-files.mdx:107]: And what about `jsonc` here?

Comment (wellwelwel) [reply website/docs/documentation/poku/config-files.mdx:107]: I couldn't find a theme for `jsonc` 😕

Comment (mrazauskas) [reply website/docs/documentation/poku/config-files.mdx:107]: Ok. Reverting

**Comment by wellwelwel** [comment]
Thanks, @mrazauskas 🤝

**Pull request merged**